### ISSUE TITLE
JUCE overwrites Makefile / Cradle

### ISSUE THREAD

**Issue by jim-lofelt** [jbloit/android_libPd_juce#1]
I know at the moment this is "by design" as you excluded the cmake and gradle file from .gitignore. However, for others to properly find the JUCE modules they will eventually need to do an export from ProJucer (which will wipe out the cmake / gradle file again). 

Perhaps there is a workaround within Projucer itself? 

For example when building for Xcode I used the "Xcode Subprojects" field in the Xcode and pointed toward the Xcode project for libpd, and selected the right target: 
**iOS** ./Libs/libpd/libpd.xcodeproj:libpd-ios
**MacOS** ./Libs/libpd/libpd.xcodeproj:libpd-osx

Unfortunately, I haven't worked with Android projects much nor the Android exporter

**Comment by jbloit** [comment]
Yes, these are Juce specifics I haven't got a chance to solve yet, although [I asked on the forum in a related thread](https://forum.juce.com/t/how-to-link-mp3lame-on-android/36520/4). So for now I still have to edit the gradle and cmake files after each projucer export. Were you able to compile the project anyway?
